components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -139,9 +142,7 @@ double GetPopRecency(const mojom::FeedItemMetadataPtr& article) {
 
   auto& publish_time = article->publish_time;
 
-  double popularity = article->pop_score == 0
-                          ? features::kBraveNewsPopScoreFallback.Get()
-                          : article->pop_score;
+  double popularity = std::min(article->pop_score, 100.0)/100.0 + features::kBraveNewsPopScoreFallback.Get();
```

Comment:
true, we are normalising over [1,100] on the server side. If the server side normalisation works as expected `std::min(article->pop_score, 100.0)/100.0` won't make a difference, but I thought it might be good to have this redundant check on the client side just to make absolutely sure this expectation is respected for the following steps. What do you think?

Reply:
Yeah should be fine :smile: